aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -951,4 +984,56 @@ protected ProgressEvent<ResourceModel, CallbackContext> versioned(
         }
         return methodVersions.get(apiVersion).invoke(proxy, rdsProxyClient.forVersion(apiVersion), progress, allTags);
     }
+
+    protected ProgressEvent<ResourceModel, CallbackContext> stopAutomaticBackupReplicationInRegion(
+            final String dbInstanceArn,
+            final AmazonWebServicesClientProxy proxy,
+            final ProgressEvent<ResourceModel, CallbackContext> progress,
+            final ProxyClient<RdsClient> sourceRegionClient,
+            final String region
+    ) {
+        final ProxyClient<RdsClient> rdsClient = proxy.newProxy(() -> new RdsClientProvider().getClientForRegion(region));
```

Comment:
Let's move this out. Apparently, we need a static method `getRdsClientForRegion(Region region)`.